karabo_bridge/tests/test_server.py

```diff
@@ -0,0 +1,28 @@
+from queue import Full
+import pytest
+
+from karabo_bridge import Client
+
+from .utils import compare_nested_dict
+
+
+def test_fill_queue(server, data):
+    for _ in range(10):
+        server.feed(data)
+
+    assert server.buffer.full()
+    with pytest.raises(Full):
+        server.feed(data, block=False)
+
+    for _ in range(10):
+        assert server.buffer.get() == (data, None)
```

Comment:
Not sure this is necessary - it looks like we're basically testing the stdlib `queue` class?

Reply:
Right